census/README.md

```diff
@@ -388,6 +388,62 @@ gcloud ml-engine jobs describe $JOB_NAME
 Once the job is `SUCCEEDED` you can check the results in `--output-path`.
 
 
+# (Optional) Preprocessing with Dataflow
+
+**Note: This is available only for estimator/.**
```

Comment:
nit: Change case to Estimator + Include Link https://github.com/GoogleCloudPlatform/cloudml-samples/blob/d8eac235c64462fedde9230a6941e09b09b82ec9/census/estimator

Reply:
Done